packages/data-context/src/data/ProjectLifecycleManager.ts

```diff
@@ -717,10 +710,8 @@ export class ProjectLifecycleManager {
     }
   }
 
-  destroy () {
-    this.resetInternalState()
-    // @ts-ignore
-    process.removeListener('exit', this.onProcessExit)
```

Comment:
Do we not need to remove this anymore?

Reply:
Nope, we no longer add the listener in the first place.